<launch>
    <node pkg="actuatorcontroller_ros" type="innfos_actuator" name="innfos_actuator" output="screen" respawn="false">
         <param name="innfos_fixed_rate" value="100" type="int"/>
    </node>
</launch>